app/src/main/java/org/simple/clinic/summary/SummaryBloodPressureListItem.kt

```diff
@@ -50,68 +52,64 @@ data class SummaryBloodPressureListItem(
     val context = holder.itemView.context
     val resources = context.resources
 
+    holder.itemView.isClickable = isBpEditable
+    holder.itemView.isFocusable = isBpEditable
+    if (isBpEditable) holder.itemView.setOnClickListener { uiEvents.onNext(PatientSummaryBpClicked(measurement)) }
```

Comment:
Won't we have to set the click listener to `null` if the bp is not editable.

Reply:
Since we are setting `isClickable` and `isFocusable` to `false`, we don't need to do this.